.prettierrc.yaml

```diff
@@ -0,0 +1,4 @@
+trailingComma: "es5"
+semi: false
```

Comment:
Maybe I'm wrong here but I consider using semicolons everywhere as a relict of the past.

Reply:
OK so overall I would still like to get rid of the semicolons because I'm in the camp of people who prefer to reduce the noise in code as much as possible and semicolons are considered by myself as noise during visual parsing.

However after investigating further it probably is best to ask the autoformatter to add semicolons while we can still code mixed style locally. I will change the option then.

Reply:
In sane languages semicolons are either mandatory or decorative, and thus can count as noise.

In JavaScript, semicolons are **optional but semantically significant**, so you're going to be much, much better off always writing them down and acting as though they were mandatory.

Reply:
To me the mechanism which automatically adds the semicolons is actually only an adjusted parser which follows parsing rules that don't require semicolons. So overall it is just necessary to understand why a specific statement is different to one without semicolons. In general it is good if one learns about these, especially because they are not that difficult (happily referring to a stackoverflow [answer](https://stackoverflow.com/a/48089473) about this topic).

Reply:
I don't understand how
```javascript
a = b + c
(d + e).foo()
```
is not a very strong argument for always having semicolons.

Reply:
It is, but knowing it is also good because it is an important detail of how fecked up javascript is after all. I mean I did decide in favour of semicolons after all because of this new information. However this still won't stop me to not use semicolons all the time while doing local development, I have a formatter which adds it after all :smile: